.github/gh-checktags.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+## for the release tags v1*, check if
+## - golang sdk tag exists
+## - spin templates tag exists
+
+set -e
+
+which gh &> /dev/null || {
+  echo "This script requires the Github CLI 'gh'"
+  exit 1
+}
+
+tags=$(gh release list --repo fermyon/spin --exclude-pre-releases --exclude-drafts | grep -v TITLE | awk '{print $1}')
+
+failed=0
```

Comment:
nit: can we just call this `exit` or `exit_code`?

Reply:
done 👍